init/init.go

```diff
@@ -168,10 +171,21 @@ func getLaunchConfig(cfg *config.CloudConfig, dockerCfg *config.DockerConfig) (*
 	return &launchConfig, args
 }
 
+func isInitrd() bool {
+	var stat syscall.Statfs_t
+	syscall.Statfs("/", &stat)
+	return stat.Type == 0x01021994
+}
+
 func RunInit() error {
 	os.Setenv("PATH", "/sbin:/usr/sbin:/usr/bin")
-	// Magic setting to tell Docker to do switch_root and not pivot_root
-	os.Setenv("DOCKER_RAMDISK", "true")
+	if isInitrd() {
+		log.Info("Booting off an in-memory FS")
```

Comment:
Can you remove this log message or move it to debug.  I don't want to see this on every boot.  And change `FS` to `file system`.